tests/__init__.py

```diff
@@ -32,6 +32,9 @@
 # Set up the default mongo binaries to use from MONGOBIN.
 set_releases(*TEST_RELEASES)
 
+# Turn off journal and preallocation for tests.
+Server.mongod_default['nojournal'] = True
+Server.mongod_default['noprealloc'] = True
```

Comment:
Without these, the tests take so long they time out and fail on my other laptop. The tests are only run with mmapv1 anyway.